src/main/java/com/philderbeast/prisonpicks/Util.java

```diff
@@ -79,6 +80,23 @@ static int calculateExperienceForBlock(Block block)
                 min = 2; 
                 max = 5; 
                 break; 
+            case DEEPSLATE_COAL_ORE:
+                min = 0; 
```

Comment:
min 0? you can get nothing from this block?

Reply:
mistake, meant to copy the normal coal ore

Reply:
then use a fall through on coal so you dont repeat your self.